app/controllers/UserController.scala

```diff
@@ -264,6 +303,7 @@ class UserController @Inject()(userService: UserService,
             lastTaskTypeIdOpt) =>
         for {
           user <- userDAO.findOne(userId) ?~> "user.notFound" ~> NOT_FOUND
+          // properties for team managers dataset manager and admins only: experiences, teams
```

Comment:
```suggestion
          // properties that can be changed by team managers and admins only: experiences, team memberships
```

(dataset managers shouldn’t have any special rights for editing users)